docs/pages/access-graph.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: Teleport Access Graph
+description: A reference for Teleport Access Graph.
+---
+
+<Admonition type="tip" title="Preview">
+  Teleport Access Graph is currently in Preview. Currently, it is only available
+  to the Teleport Cloud Team users. Not all Teleport resources are currently
```

Comment:
Does "Teleport Cloud Team" mean both "Teleport Enterprise Cloud" and "Teleport Team"?